api/models/Client.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+public class Client : BaseModel
```

Comment:
Nu dat de tabel client tbl gaat bestaan in de datbase kunnen we de `Orders` linken aan een client. Zorg er alleen nog voor dat de relatie tussen die 2 in models terecht komen en daarvoor een migration maken.

![image](https://github.com/user-attachments/assets/23d9a2e0-985c-487e-85d7-099243cd9918)